cogs/wiki.py

```diff
@@ -266,6 +266,15 @@ async def unlaunchbg(self, ctx):
         embed.url += "twilightmenu/custom-unlaunch-backgrounds.html"
         embed.description = "How to make custom Unlaunch backgrounds and install them using TWiLight Menu++"
         await ctx.send(embed=embed)
+        
+            @commands.command(aliases=["hiya", "hiyacfw"])
```

Comment:
this has too many spaces for whatever reason. It should be in line with the line right below.